### DISCUSSION TITLE
How do I edit column type in SQLite ?

### DISCUSSION THREAD

**Discussion by dittorahmat** [nocodb/nocodb#969] [Q&A] [answered]
How do I edit column type in SQLite ? I want to change a column type to URL. It should be possible looking from the docs

https://docs.nocodb.com/setup-and-usages/tables

However, I can only rename it. The rest was showing warning "This spreadsheet is connected to an SQLite DB. For production please see ."

I'm using this for a small project with 1 table only, so deploying mysql db might be an overkill.

**Comment by pranavxc** [comment, marked as answer]
SQLite only supports limited support of `alter table` statement so in order to avoid problems while altering column type we are disabled the column edit option in SQLite.

Reference : https://stackoverflow.com/a/4820137
